agent/csi/libstorage/csi_service_libstorage.go

```diff
@@ -486,10 +502,16 @@ func (d *driver) NodePublishVolume(
 	}
 
 	if mv := req.VolumeCapability.GetMount(); mv != nil {
+		if !st.IsDir() {
```

Comment:
Hi @codenrhoden,

Nevermind, I see why you didn't use gotil. You're using the result of the stat call here. Thanks.